apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -24,6 +24,13 @@ export default class CoreLibrary {
     this.eventLog = [];
     this.speechBubbles = [];
     this.soundLog = [];
+    this.criteria = [];
+    this.previous = {};
+    this.validationTimes = {
+      early: 30,
+      wait: 150,
+      delay: 0
```

Comment:
What unit of time are we using for these, e.g. seconds, frames?  Can we use named constants or at least comments explaining?